src/prisma/migrations/20231129003148_add_space_settings/migration.sql

```diff
@@ -0,0 +1,2 @@
+-- AlterEnum
+ALTER TYPE "ProposalEvaluationType" ADD VALUE 'pass_fail';
```

Comment:
Overall suggestion; (if not a burden) merging all these migrations into one file

Reply:
Yes! definitely.. that was my plan. I was making lots of tweaks so i gave up trying until I'm finished. I'll go ahead and merge the ones I've got